package.json

```diff
@@ -74,13 +81,15 @@
     }
   ],
   "eslintConfig": {
-    "extends": "@logux/eslint-config",
+    "extends": "@logux/eslint-config/ts",
     "rules": {
       "security/detect-non-literal-regexp": "off",
       "security/detect-unsafe-regex": "off",
       "node/no-missing-require": "off",
       "no-unused-expressions": "off",
-      "no-unused-vars": "off"
+      "no-unused-vars": "off",
+      "no-redeclare": "off",
+      "@typescript-eslint/no-redeclare": ["error"]
```

Comment:
```suggestion
      "@typescript-eslint/no-redeclare": "error"
```

Reply:
What error it prevents in Autoprefixer?

Reply:
`no-redeclare` rule forbid to make function overloading. Maybe it is better to move rules to `@logux/eslint-config`:
```
      "no-redeclare": "off",
      "@typescript-eslint/no-redeclare": "error"
```

Reply:
Where it could happen?

Reply:
In any place when you will have function overloading or interface declaration merging. `no-redeclare` doesn't understand that it is not "redeclare", `@typescript-eslint/no-redeclare` handles these cases.

Reply:
Thanks for the suggestion. I released ESLint config 44.0.1 and updated config in Autoprefixer’s `master`.

Can you revert config changes (and we are ready for the release)?